browser/profiles/brave_renderer_updater.cc

```diff
@@ -130,8 +130,7 @@ bool BraveRendererUpdater::CheckActiveWallet() {
   if (!keyring_service_) {
     return false;
   }
-  bool is_wallet_created = keyring_service_->IsKeyringCreated(
-      brave_wallet::mojom::kDefaultKeyringId);
+  bool is_wallet_created = keyring_service_->IsWalletCreatedSync();
```

Comment:
this is actually an anti-pattern here for prefs that I brought up recently in the dev channel. It creates a dependency for wallet on the KeyringService that shouldn't be necessary here. I would pass the raw value of the pref and have a helper in common to replace `KeyringService::IsWalletCreatedSync`